pm_symmetry/test-data/test_out.txt

```diff
@@ -0,0 +1,52 @@
+Wyckoff points symmetry report for input.cell
```

Comment:
This one is pedantic, but it would be nice if you could get `input.cell` in these outputs to be replaced by the input filename - i.e. `Si.cell` in this case. Prevents confusion.

Reply:
Actually, thinking about it, perhaps we don't even need to use the `input.cell` placeholder in the tool command - just symlink to `$structure.name`? Perhaps you can skip the symlink altogether, but I don't know if that would result in some long absolute path being output, which we'd want to avoid.

Reply:
updated to use $structure.name